model/project.go

```diff
@@ -1430,6 +1430,16 @@ func (p *Project) GetModuleByName(name string) (*Module, error) {
 	return nil, errors.New("no such module on this project")
 }
 
+func (p *Project) GetAutoUpdateModules() ModuleList {
```

Comment:
nit: function comment (I know we don't have these for all exported functions but we should try to add them to keep up with the style guide

Reply:
🤦‍♂️ sure thing!